epoxy-adapter/build.gradle

```diff
@@ -1,26 +1,35 @@
 apply plugin: 'com.android.library'
+apply from: "$rootDir/blessedDeps.gradle"
```

Comment:
I tried that but was getting a `variable not found` error for some odd reason. I will try once again since your refactor may have solved it :+1: